content/docs/reference-events.md

```diff
@@ -6,13 +6,13 @@ layout: docs
 category: Reference
 ---
 
-This reference guide documents the `SyntheticEvent` wrapper that forms part of React's Event System. See the [Handling Events](/docs/handling-events.html) guide to learn more.
+यह संदर्भ React इवेंट सिस्टम का हिस्सा बनने वाले `SyntheticEvent` आवरण को निर्देशित करता है। अधिक जानने के लिए [Handling Events](/docs/handling-events.html) गाइड देखें।
```

Comment:
यह संदर्भ React इवेंट सिस्टम का हिस्सा बनने वाले `SyntheticEvent` आवरण को निर्देशित करता है। -> could be simplified a bit? too complex

Reply:
आवरण could be wrapper in hindi